code/dynamic_programming/src/rod_cutting/README.md

```diff
@@ -1,4 +1,33 @@
-# cosmos
-Your personal library of every algorithm and data structure code that you will ever encounter
+# Rod Cutting
+Given a rod of length n units and an array of prices p<sub>i</sub> for rods of length i = 1, 2, . . . ,n. Determine the maximum revenue r <sub>n</sub> obtainable by cutting the rod and selling the pieces.
+
+**For Example -**
+
+Let the table of prices be - 
+
+length (i) | 1|2|3|4|5|6|7|8|9|10  
+-----------|--|-|-|-|-|--|-|--|--|------------  
+price (p<sub>i</sub>)|1|5|8|9|10|17|17|20|24|30
+
+For a rod of length n = 4, maximum revenue is obtained by cutting the rod into two pieces each of length 2 and the maximum revenue obtained is 10.
+
+ 
+## Explanation
+This problem satisfies both properties of Dynamic programming and can be solved by the following recurrence relation.
+
+
```

Comment:
You forgot to list an algorithm as a list of steps according to [our guide](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md).